src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java

```diff
@@ -219,6 +224,7 @@ public ComplaintResource(AuthorizationCheckService authCheckService, ExerciseRep
      * @param principal that wants to get complaints
      * @return the ResponseEntity with status 200 (OK) and a list of complaints. The list can be empty
      */
+    @Deprecated // will be replace with a method which returns the submissions
```

Comment:
The comment can be added as String to the tag to make it easier available

Reply:
I deleted the method